### ISSUE TITLE
CAD files

### ISSUE THREAD

**Issue by mdvorak** [moggieuk/ERCF-Springy#2]
Hi, this looks really great, would you mind releasing CAD files? I'd like to do some minor modifications so I can use what I have already at home.

Thanks!

Side note: Is there a reason, why heatset insert for the spring screw is burried deep in the hole, instead simply putting it on top? It would be way easier to put it there.

**Comment by ntchris** [comment]
CAD file would be nice, sure.
for the heat insert, my understanding is, if put it on top, it could be easily pushed out by the strong spring.  but like currently put in on the other side,  it's very strong, can resist the strong spring 's force upward.

**Comment by mdvorak** [comment]
Good point, I don't think properly positioned heatinsert would actually fall out (especially when pushed a little further under the surface), but I trust @moggieuk :)

**Comment by todstelzer** [comment]
Just wanted to do a minor chance on the top hat and there are no CAD or STEP files...
Why?
